### PR TITLE
Azure DNS Component

### PR DESCRIPTION
**Pull request by DevSecNinja**
**Description:**

With the Azure DNS component, users can keep their external IPv4 address in sync with an Azure DNS record

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21366

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/azuredns.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## {% linkable_title Configuration %}
+
+To use the component in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+azuredns:
+  domain: example.com
+  host: hass
```
Comment (klaasnicolaas) [inline source/_components/azuredns.markdown]: If `host` is not required you can remove it

File: source/_components/azuredns.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+date: 2019-02-23 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: azure.png
+ha_category: Network
+featured: false
+ha_release: 0.90
+---
```
Comment (klaasnicolaas) [inline source/_components/azuredns.markdown:14]: What is the `ha_iot_class`?

Comment (klaasnicolaas) [reply source/_components/azuredns.markdown:14]: https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/azuredns.markdown (hunk 2 above)
Comment (DevSecNinja) [reply source/_components/azuredns.markdown:14]: Good one @klaasnicolaas. This component is not communicating with an IoT device, so I'm not sure if this needs to be specified. I've checked the [Namecheap component](https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/namecheapdns.markdown) and the [DuckDNS component](https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/duckdns.markdown) which offers similar functionalities, and I don't see the `ha_iot_class` specified there.

Can you confirm if this is required? Thanks.

File: source/_components/azuredns.markdown (hunk 1 above)
Comment (DevSecNinja) [reply source/_components/azuredns.markdown]: Thanks, good one. Fixed in 8c8d1084e0d08ed0332df7c593a65607a771f243.

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**